.github/CONTRIBUTING.md

```diff
@@ -27,8 +26,11 @@ If you're not used to this workflow with git, you can start with some [docs from
 
 ## Tests
 
-You can optionally test your changes by running the pipeline locally. Then it is recommended to use the `debug` profile to
-receive warnings about process selectors and other debug info. Example: `nextflow run . -profile debug,test,docker --outdir <OUTDIR>`.
+You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile. Execute all the tests with the following command:
+
+```bash
+nf-test test --profile debug,test,docker --verbose
```

Comment:
Does thus command work for this repo currently? I think you need some more setup of the repo in order to test the pipeline with nf-test. I get the following error:
```shell

🚀 nf-test 0.8.4
https://code.askimed.com/nf-test
(c) 2021 - 2024 Lukas Forer and Sebastian Schoenherr

Warning: This pipeline has no nf-test config file.
Error: No tests or test directories containing scripts that end with *.test provided.
```

Reply:
Updated in ffed4c05c54ec05345a4b1b8cb0506f7aa05de4b, 255d91ca95acf9785c6b30f680faee7eef3b4160 and 2c1f1e5c91ace13c2b0448086e627ae8061839a7